src/observe.js

```diff
@@ -1685,7 +1685,8 @@
 
   var expose = global;
 
-  if (typeof exports !== 'undefined') {
+  if (typeof exports !== 'undefined' && exports + '' === '[object Object]' &&
```

Comment:
ah, nice find. yeah that's some intense checking.
BTW, I'm happy to accept this as is too. Just wondering how we can best harden it. You'd think there would be a "standard" way to do this by now :)